resources/js/Pages/Results.vue

```diff
@@ -73,9 +72,9 @@
                 :id="`item-mismatches-${String(item)}`"
                 :key="idx">
                 <h2 class="h4">
-                    <wikit-link :href="`https://www.wikidata.org/wiki/${String(item)}`" target="_blank">
+                    <a :href="`https://www.wikidata.org/wiki/${String(item)}`" target="_blank">
```

Comment:
Adding an anchor element inside a header collapses the margins, this doesn't happen with the wikit-link element.

See:
with anchor (considerably smaller vertical margin)
![image](https://github.com/wmde/wikidata-mismatch-finder/assets/3005239/606d49ef-8f22-4e11-899b-899a5d8e1926)

original (wikit-link)
![image](https://github.com/wmde/wikidata-mismatch-finder/assets/3005239/c83ce1b3-48a9-4fad-946c-eaaab1d90cf1)

Reply:
good catch, I accidentally moved some styles that applied on wikit-link to the containing element instead of having them on `<a>`